tests/admin_views/tests.py

```diff
@@ -924,6 +924,15 @@ def test_resolve_admin_views(self):
         self.assertIs(index_match.func.admin_site, customadmin.simple_site)
         self.assertIsInstance(list_match.func.model_admin, customadmin.CustomPwdTemplateUserAdmin)
 
+    def test_resolve_admin_urls(self):
```

Comment:
This is probably better in `tests/modeladmin`. How about a test for `AdminSite.admin_index_url()`?

The tests are lacking in the fact that there's nothing to guarantee these methods are used throughout the admin views and templates. I guess that would substantial testing.